### ISSUE TITLE
ERR_INVALID_ARG_TYPE when importing lib

### ISSUE THREAD

**Issue by shobhitsrivastava** [puppeteer/puppeteer#5794]
* Puppeteer version: **3.0.2**
* Platform / OS version: **Windows 10**
* Node.js version: **v8.9.4**

Just getting started with Puppeteer, this import line: `const puppeteer = require('puppeteer');`

causes this issue: 
```
internal/util.js:209
    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'original', 'function');
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "original" argument must be of type function
    at promisify (internal/util.js:209:11)
    at Object.<anonymous> (c:\Users\Shobhit\Documents\lnkdn\node_modules\extract-zip\index.js:11:18)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (c:\Users\Shobhit\Documents\lnkdn\node_modules\puppeteer\lib\BrowserFetcher.js:25:17)
```
I don't have a similar issue in my environment importing other npm libraries.

**Comment by shobhitsrivastava** [comment]
fixed by updating node from v8.9.4 -> v12.16.3.

**Comment by javanf** [comment]
同上
